docs/ug/usingReports.md

```diff
@@ -140,6 +139,11 @@ The `Code panel` allows users to see the code attributed to a specific author. C
 * Code attributed to the author is highlighted in green.
 * Non-trivial code segments that are not written by the selected author are hidden by default, but you can toggle them by clicking on the %%:fas-plus-circle:%% icon.
 
+<box type="info" seamless>
+
+Changing of binary files, such as png and jpg files, is not counted towards the total number of lines of code contributed by an author.
```

Comment:
```suggestion
Changing binary files, such as png and jpg files, is not counted towards the total number of lines of code contributed by an author.
```

Reply:
Thanks for pointing it out. I have edited it accordingly. Is there any further comment on the documentation part?

Reply:
In "Troubleshooting" in the UG we say that we don't analyze such files, so that needs to be removed and the documentation regarding the files we analyze needs to be updated accordingly.

Reply:
I have removed that section. I also changed the documentation regarding the formats flag and the File formats argument in the repo csv file, since adding extensions such as `png` and `jpg` will include those files grouped under the file type `Binary`